src/components/Discipline.vue

```diff
@@ -57,6 +57,16 @@ export default {
         )
         .map(({ id }) => id);
     },
+    computeBonus() {
```

Comment:
mozno zvolit ine meno, je to sice computed, ale nemusi sa tam pouzivat to klucove slovo,
druha vec je v tom nazvu sloveso co sa pouziva viacmenej u funkcii (asi by sa dalo computed pocitat ako funkcia, ale ja to berem ako premennu alebo konstantu) tak mozno lepsi nazov by mohol stacit len `bonus` ak sa to teda nebije s nejakou prop alebo niecim (ak hej tak asi nechaj tak, moze sa to riesit inokedy)